src/pages/OnboardEngagement/ExpensifyClassicPage.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {ScrollView, View} from 'react-native';
+import Button from '@components/Button';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import * as DeviceCapabilities from '@libs/DeviceCapabilities';
+import Navigation from '@libs/Navigation/Navigation';
+import variables from '@styles/variables';
+import * as Link from '@userActions/Link';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function ExpensifyClassicModal() {
+    const StyleUtils = useStyleUtils();
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const {isExtraSmallScreenHeight, windowHeight} = useWindowDimensions();
+    const canUseTouchScreen = DeviceCapabilities.canUseTouchScreen();
+    const theme = useTheme();
+    const iconWrapperHeight = windowHeight / variables.oldDotWireframeIconWrapperHeightFactor;
+
+    const navigateBack = () => {
+        Navigation.goBack(ROUTES.ONBOARD_MANAGE_EXPENSES);
+    };
+
+    const navigateToOldDot = () => {
+        Link.openOldDotLink(CONST.OLDDOT_URLS.DISMMISSED_REASON);
```

Comment:
Looks good - we are going to want the full link to also go to inbox like this 

`https://expensify.com/inbox?dismissedReason=missingFeatures`

Reply:
Sorry I could have been more clear about that, thanks!

Reply:
alright, will update.

Reply:
Thanks